### PR TITLE
Add attemps to login

### PR DESCRIPTION
**Pull request by koval1off**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -48,15 +58,21 @@ def menu():
...
-        return
-
-    print('Welcome to our Bank!')
-    menu()
+    while True:
+        user_id = get_number_from_input("Enter your ID: ")
+        user_pass = get_number_from_input("Enter you password: ")
+        try:
+            if not check_id_and_pin(user_id, user_pass):
+                print(f"Something wrong with your ID or PASS. {count}/3. Try again")  # need not to TRY AGAIN at the last time
```
Comment (koval1off) [inline main.py]: if it's needed, I can try to not print TRY AGAIN at the last try

File: main.py [hunk 2]
```diff
@@ -15,6 +15,16 @@ def get_number_from_input(text):
     return int(num)
 
 
+count = 1
+def count_attemps():
+    global count
```
Comment (koval1off) [inline main.py]: I'm using GLOBAL here, but you're asked me not to do that

File: main.py [hunk 3]
```diff
@@ -15,6 +15,16 @@ def get_number_from_input(text):
     return int(num)
 
 
+count = 1
+def count_attemps():
```
Comment (koval1off) [inline main.py]: may be this function must be in class ATM. And than I can try to do same things , without using global
and may be it's will be more correctly . idk

File: main.py [hunk 4]
```diff
@@ -15,6 +15,16 @@ def get_number_from_input(text):
...
+    global count
+    if count < 3:
+        count += 1
+    else:
+        quit()
```
Comment (Alex-Krykun) [inline main.py]: DO NOT USE QUIT
https://pythonguides.com/python-exit-command/#:~:text=exit()%20commands.-,Python%20quit()%20function,be%20used%20in%20the%20interpreter.

File: main.py (hunk 2 above)
Comment (Alex-Krykun) [reply main.py]: U may use local variable inside main() function

File: main.py [hunk 5]
```diff
@@ -48,15 +58,21 @@ def menu():
...
+        user_pass = get_number_from_input("Enter you password: ")
+        try:
+            if not check_id_and_pin(user_id, user_pass):
+                print(f"Something wrong with your ID or PASS. {count}/3. Try again")  # need not to TRY AGAIN at the last time
+                count_attemps()
+                continue
+            else:
+                print('Welcome to our Bank!')
+                menu()
+                break
```
Comment (Alex-Krykun) [inline main.py]: why you need break here?

File: main.py [hunk 6]
```diff
@@ -48,15 +58,21 @@ def menu():
...
+        except:
+            print("Try next day!")
+            break
```
Comment (Alex-Krykun) [inline main.py:61]: why you need break here?

File: main.py [hunk 7]
```diff
@@ -48,15 +58,21 @@ def menu():
...
-    user_id = get_number_from_input("Enter your ID: ")
-    user_pass = get_number_from_input("Enter you password: ")
-
-    if not check_id_and_pin(user_id, user_pass):
-        print("Something wrong with your ID or PASS. Try again")
-        return
-
-    print('Welcome to our Bank!')
-    menu()
+    while True:
```
Comment (Alex-Krykun) [inline main.py]: do we need while true if only 3 attempts?
do we need a while for the whole code or only for reading pin/id?

File: main.py (hunk 5 above)
Comment (Alex-Krykun) [reply main.py]: if you fix while, you will not need this

File: main.py (hunk 6 above)
Comment (Alex-Krykun) [reply main.py:61]: if you fix while, you will not need this

File: .idea/workspace.xml [hunk 8]
```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Alex-Krykun) [inline .idea/workspace.xml]: YOU DON:T NEED TO COMMIT THIS FILE

Comment (Alex-Krykun) [reply .idea/workspace.xml]: the same for __pycache__/bank.cpython-39.pyc

**Review by Alex-Krykun — APPROVED** [review]

**Pull request merged**